### PR TITLE
Build doc

### PR DESCRIPTION
**Pull request by RKuttruff**
Documentation for building the Docker images used by SDAP.

### CODE REVIEW

File: docs/build.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+.. code-block:: bash
+
+  cd docker/solr
+
+Now to build the image:
+
+.. code-block:: bash
+
+  docker build . -t sdap/solr:${NEXUS_VERSION}
```
Comment (ngachung) [inline docs/build.rst:95]: For consistency with public docker images, this should be named sdap/solr-cloud

**Review by ngachung — CHANGES_REQUESTED** [review]
In quickstart line 37 "NEXUS JPL repository <https://hub.docker.com/u/nexusjpl>" should point to https://hub.docker.com/search?q=apache%2Fsdap

File: docs/build.rst (hunk 1 above)
Comment (RKuttruff) [reply docs/build.rst:95]: Should I prefix all the images with `sdap-` as well?

Comment (ngachung) [reply docs/build.rst]: Maybe not since we're using sdap/ whereas in the apache official images we have apache/sdap-

Comment (RKuttruff) [reply docs/build.rst]: Though it would be a bit easier to use with the qs since you'd only have to change the repo instead of repo + image

Comment (ngachung) [reply docs/build.rst]: True, how about local/sdap-nexus-webapp to distinguish between pulling official image from docker hub?

Comment (RKuttruff) [reply docs/build.rst]: Or I can make it a variable in the quickstart
ie: 
`export REPO=apache/sdap-` for official
vs
`export REPO=sdap/` for local

And then refer to the images with something like `${REPO}solr-cloud`

**Review by ngachung — APPROVED** [review]

**Pull request merged**